packages/core/src/components/Modal/ModalHeader/ModalHeader.tsx

```diff
@@ -12,6 +12,7 @@ import Text from "../../Text/Text";
 import Heading from "../../Heading/Heading";
 import Flex from "../../Flex/Flex";
 import styles from "./ModalHeader.module.scss";
+import { SubIcon } from "src/types";
```

Comment:
I'm not sure how it behaves
Can you, for the meantime, change it to a relative path? same as other imports in the file? (`../../../types`)

Reply:
Sure @YossiSaadi, allow me some time I'll raise another PR.